ts/components/session/SessionRegistrationView.tsx

```diff
@@ -6,6 +6,7 @@ import { SessionIconButton, SessionIconSize, SessionIconType } from './icon';
 
 export const SessionRegistrationView: React.FC = () => (
   <div className="session-content">
+    <div id="session-toast-container" />
```

Comment:
Purpose of moving the toast container into here?

Reply:
We didn't have it in the session registration view. So, toasts made before the registration is complete are not shown. And I need to show a toast during registration.

Reply:
Oh, great -- sounds good.